states.inc.php

```diff
@@ -92,7 +92,7 @@
         "name" => "loadAnimal",
         "description" => clienttranslate('${actplayer} must load an animal'),
         "descriptionmyturn" => clienttranslate('${you} must load an animal'),
-        "descriptionimpossible" => clienttranslate('${actplayer} must load an animal'),
+        "descriptionimpossible" => clienttranslate('${actplayer} must take back all animals present on the ferry'),
```

Comment:
This is wanted, as opponent always see the same message, only the active player see the real notice